Flavor nigeria dcis

qaziabubakar-vd opened the pull request:
Introduce new flavor into WCARO with the following changes mentioned in the data dictionary:
https://drive.google.com/file/d/1qwchfIxcdYRchlvd_lELiSNSk9SvLYYj/view?ts=608aa19c

NIGERIA Board:
https://github.com/orgs/opensrp/projects/48

qaziabubakar committed:
Added new flavor 'nigeria' to the existing chw project

qaziabubakar committed:
Updated authenticator.xml to use chad authentication for testing

qaziabubakar committed:
Added new forms for nigeria flavor

qaziabubakar committed:
Updated build.gradle

zzainulabidin committed:
Merge branch 'master' into flavor-nigeria-dcis

qaziabubakar committed:
Added new forms for Nigeria DCIS flavor

qaziabubakar committed:
Added new family_member_register form

qaziabubakar committed:
Updated vaccines for child home visit

qaziabubakar committed:
Updated family-member-relevance.yml

zzainulabidin committed:
Merge pull request #1776 from opensrp/new-forms-nigeria-dcis

Added new forms for Nigeria DCIS flavor

zzainulabidin committed:
Add nigeria updates

zzainulabidin committed:
Merge pull request #1781 from opensrp/updates-for-nigeria

Updates for Nigeria Flavor

zzainulabidin committed:
Add wash_check checks for nigeria flavor

zzainulabidin committed:
Change login icon for login screen

zzainulabidin committed:
add support for disability; fix pregnancy outcome translation issue

zzainulabidin committed:
Update anc_pregnancy_outcome.json

zzainulabidin committed:
Add required checks for child disability

zzainulabidin committed:
Update .gitignore

zzainulabidin committed:
update gradle for kujaku lib import

zzainulabidin committed:
Merge pull request #1788 from opensrp/nigeria-child-disability-in-home-visit

Add child disability, Bug fixing

zzainulabidin committed:
Fix translation

zzainulabidin committed:
Add missing images into main dir

vincent-karuri commented:
@qaziabubakar-vd please link to the issues this PR closes.

Also, the build is not running, could you look into that?

zzainulabidin committed:
Working on Client feedback after taning

zzainulabidin committed:
Update family_register form

zzainulabidin committed:
Update family_member_register json

zzainulabidin committed:
resolve fileds persistent issues

zzainulabidin committed:
resolve family_member_register fields persistent issues in edit

zzainulabidin committed:
fixes for child_disability visit form

zzainulabidin committed:
Merge branch 'master' into flavor-nigeria-dcis

zzainulabidin committed:
Implement location hierarchy

zzainulabidin committed:
update family_details_register for nigeria

zzainulabidin committed:
Update vaccines.json

zzainulabidin committed:
Add income bracket question in add family

zzainulabidin committed:
Add exit dialog strings

zzainulabidin committed:
Fix Washcheck sync issue and fix take photo crash

zzainulabidin committed:
Update DefaultChildProfileActivityFlv

zzainulabidin committed:
Update build.gradle

zzainulabidin committed:
Merge branch 'master' into flavor-nigeria-dcis

zzainulabidin committed:
Update chw-core SNAPSHOT version

zzainulabidin committed:
Implement codacy review

zzainulabidin committed:
Make wash check non-editable

zzainulabidin committed:
fix error for empty value

zzainulabidin committed:
implement codacy feedback

zzainulabidin committed:
Resolve code complexity

zzainulabidin committed:
Fix codecy issue

zzainulabidin committed:
Merge branch 'master' into flavor-nigeria-dcis

zzainulabidin committed:
Add washcheck unit test

allan-on committed:
:construction: Enable sync progress bar

allan-on committed:
:construction: Add force sync improvements

allan-on committed:
:arrow_up: Update chw-core snapshot and app versions

zzainulabidin committed:
Merge branch 'master' into flavor-nigeria-dcis

zzainulabidin committed:
Write unit tests for FamilyProfileActivityFragment

zzainulabidin committed:
Fix fam_source_income and income_bracket issue when user select no option

zzainulabidin committed:
Update login icon

zzainulabidin committed:
Merge branch 'master' into flavor-nigeria-dcis

zzainulabidin committed:
Update login screen design

zzainulabidin committed:
Update app icon

hilpitome committed:
refactored opensrp urls

hilpitome committed:
Merge branch 'flavor-nigeria-dcis' of github.com:opensrp/opensrp-client-chw into flavor-nigeria-dcis

zzainulabidin committed:
Update add family and wash check form

zzainulabidin committed:
Merge branch 'master' into flavor-nigeria-dcis

zzainulabidin committed:
Merge branch 'master' into flavor-nigeria-dcis

zzainulabidin committed:
Update Location Hierachy to Sub-Community level

zzainulabidin committed:
Merge branch 'master' into flavor-nigeria-dcis

ndegwamartin committed:
Google Playstore release
- Update compileSDK and targetSDK to target level 30
- Bump up version code and version name

ndegwamartin committed:
Merge pull request #1940 from opensrp/v1.0.3-release-ps

Google Playstore release : Nigeria DCIS

zzainulabidin committed:
Merge branch 'master' into flavor-nigeria-dcis

allan-on committed:
:arrow_up: Udpdate app version

allan-on committed:
:bug: Fix WashCheck visits NULL pre-processed value

allan-on committed:
:bug: Add migration to update visit pre_processed values

allan-on committed:
:arrow_up: Update DCIS app version
